diff --git a/CMakeLists.txt b/CMakeLists.txt
index 4ee6812..3f52d0a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,6 +121,12 @@ endif()
 
 include(CMakePackageConfigHelpers)
 
+export(
+  EXPORT ${PROJECT_NAME}-targets
+  FILE "${CMAKE_CURRENT_BINARY_DIR}/cmake/${PROJECT_NAME}-targets.cmake"
+  NAMESPACE ${PROJECT_NAME}::
+)
+
 install(
   EXPORT ${PROJECT_NAME}-targets
   NAMESPACE ${PROJECT_NAME}::
